test/conftest.py

```diff
@@ -80,10 +80,10 @@ def mock_get_current_user(mocker):
     user = User(username='bob',
                 hashed_password='$2b$12$CpJZx5ooxM11bCFXT76/z.o6HWs2sPJy4iP8.'
                                 'xCZGmM8jWXUXJZ4K',
-                active=True)
+                active=True, is_admin=False)
```

Comment:
I think this commit should have been after the one to add `is_admin` or it'll always fail.  But that's a minor thing, the tests are passing at the end of the series of commits.

Reply:
I have reordered the commits.